src/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// signature_pad's props 
+// (taken from https://github.com/szimek/signature_pad/blob/e2af461bca7ceb7cc5afb8349930908ca19f2f56/src/signature_pad.ts#L23)
```

Comment:
should we import `signature_pad`'s types directly from `signature_pad`? I haven't updated to the `beta` yet (as it's beta) so that might not be possible until then and this might be fine in the meantime